classes/classes/BodyParts/BodyPart.as

```diff
@@ -0,0 +1,47 @@
+/**
+ * Created by aimozg on 25.04.2017.
+ */
+package classes.BodyParts {
+import classes.CoC_Settings;
+import classes.Creature;
+import classes.internals.SimpleJsonable;
+
+public class BodyPart extends SimpleJsonable {
+	include "../../../includes/appearanceDefs.as";
```

Comment:
@brrritssocold 
I'm against just moving them into some class because it will make the existing codebase even more unreadable. 
I'd like to:
1. Limiting their use to respective BodyPartClass and use helper methods in the other places where more than one magic constant is present (i.e. `player.face.type==FACE_HUMAN` is OK, but `(p.f.t==F_A || p.f.t==F_B || ...)` should be replaced with `player.face.isABC`.
2. Move them as BodyPartClasses constants so that they could be used without namespace prefix there.

But there's no consensus about that so things are staying as they are.